### PR TITLE
AV-173914 update max version of avi controller

### PR DESCRIPTION
**Pull request by arihantg**

### CODE REVIEW

**Comment by arihantg** [comment]
build ako

File: internal/rest/avi_evh_obj.go [hunk 1]
```diff
@@ -519,6 +520,7 @@ func (rest *RestOperations) AviVsChildEvhBuild(vs_meta *nodes.AviEvhVsNode, rest
 			Path: &path_match,
 		}
 		vHMatchRule := &avimodels.VHMatchRule{
+			Name:    &name,
```
Comment (akshayhavile) [inline internal/rest/avi_evh_obj.go:523]: @arihantg : Is `NAME` unique key? In case of multiple MatchRule it might create problem.

Comment (arihantg) [reply internal/rest/avi_evh_obj.go:523]: there will only be a single matchrule for a hostname.So we can keep the value hardcoded

Comment (akshayhavile) [reply internal/rest/avi_evh_obj.go:523]: It will create problem for hostrule with GSLB FQDN or alias feature.

Comment (swathinsankaran) [reply internal/rest/avi_evh_obj.go:523]: @arihantg Have you tested the hostrule aliases and GSLB FQDN?

Comment (arihantg) [reply internal/rest/avi_evh_obj.go:523]: I have tested this test  test_hostrule_fdqn_aliases in test_host_rules which is passing successfully.

**Review by akshayhavile — APPROVED** [review]

File: internal/rest/avi_evh_obj.go (hunk 1 above)
Comment (swathinsankaran) [reply internal/rest/avi_evh_obj.go:523]: Thanks @arihantg

**Review by swathinsankaran — APPROVED** [review]

**Pull request merged**